### PR TITLE
GeoServer 2.24-RC release announcement

### PR DESCRIPTION
**Pull request by aaime**
Work in progress 2.24-RC announcement

### CODE REVIEW

**Comment by jodygarnett** [comment]
## printing module 

Added a bunch of new functionality from GeoSolutions and can now be used out-of-the-box by GeoNode and MapStore:

[Max number of columns configuration for multi column legends](https://github.com/geosolutions-it/mapfish-print/wiki/Max-number-of-columns-configuration-for-multi-column-legends)
[Simple colored box icon in legends](https://github.com/geosolutions-it/mapfish-print/wiki/Simple-colored-box-icons)
[Explicit support of Geoserver CQL_FILTER parameter (also with layers merge support)](https://github.com/geosolutions-it/mapfish-print/wiki/Explicit-support-of-Geoserver-CQL_FILTER-parameter)
[Legend fitting](https://github.com/geosolutions-it/mapfish-print/wiki/Legend-fitting)
[Don't break legend items](https://github.com/geosolutions-it/mapfish-print/wiki/Don't-break-legend-items)
[Reorder legends block in columns](https://github.com/geosolutions-it/mapfish-print/wiki/Reorder-legends-block-in-columns)
[Images content](https://github.com/geosolutions-it/mapfish-print/wiki/Images-content)
[Dynamic images page](https://github.com/geosolutions-it/mapfish-print/wiki/Dynamic-images-page)
[Multipage legends](https://github.com/geosolutions-it/mapfish-print/wiki/Multipage-legends)
[Custom intervals in ScalebarBlock](https://github.com/geosolutions-it/mapfish-print/wiki/Custom-intervals-in-ScalebarBlock)
[Clustering Support](https://github.com/geosolutions-it/mapfish-print/wiki/Clustering-Support)
[HTML rendering in text blocks](https://github.com/geosolutions-it/mapfish-print/wiki/HTML-In-Text-Blocks)
[Extra Pages](https://github.com/geosolutions-it/mapfish-print/wiki/Extra-Pages)
[Group Rendering in attribute blocks](https://github.com/geosolutions-it/mapfish-print/wiki/Group-Rendering-In-Attribute-Blocks)
[Skip rendering of pages](https://github.com/geosolutions-it/mapfish-print/wiki/Skip-Rendering-Of-Pages)
[Automatic X-Forwarded-For](https://github.com/geosolutions-it/mapfish-print/wiki/X-Forwarded-For)
[Parsing of Base64 encoded images](https://github.com/geosolutions-it/mapfish-print/wiki/Base64-encoded-images)

File: _posts/2023-09-25-geoserver-2-24-RC-released.md [hunk 1]
```diff
@@ -175,6 +161,22 @@ Those interested in this work are encouraged to contact Andrea Aime (GeoSolution
 * [GEOS-10909](https://osgeo-org.atlassian.net//browse/GEOS-10909) Don't link from OGC API Features to WFS 2.0 DescribeFeatureType output, if WFS is disabled
 * [GEOS-10954](https://osgeo-org.atlassian.net//browse/GEOS-10954) Split ogcapi community module package into single functionality packages
 
+### DataDir Catalogue loader
+
+For folks working with very large catalogues some improvement from [cloud native geoserver](https://github.com/geoserver/geoserver-cloud) are now available to reduce startup time.
+
+Thanks to Gabriel Roland for folding this improvement into a community module for the rest of the GeoServer community to enjoy.
```
Comment (mprins) [inline _posts/2023-09-25-geoserver-2-24-RC-released.md:168]:
```suggestion
Thanks to Gabriel Roldan for folding this improvement into a community module for the rest of the GeoServer community to enjoy.
```

File: _posts/2023-09-25-geoserver-2-24-RC-released.md [hunk 2]
```diff
@@ -175,6 +161,22 @@ Those interested in this work are encouraged to contact Andrea Aime (GeoSolution
...
+
+* [GEOS-11049](https://osgeo-org.atlassian.net//browse/GEOS-11049) Community module "datadir catalog loader"
+
+### GeoServer Access Control List Project
+
+The [GeoServer Access Control List](https://github.com/geoserver/geoserver-acl) project is an independent application service that manages access rules, and a GeoServer plugin that requests authorization limits on a per-request basis.
+
+Gabriel Roldand is the contact point for anyone interested in this work.
```
Comment (mprins) [inline _posts/2023-09-25-geoserver-2-24-RC-released.md:176]:
```suggestion
Gabriel Roldan is the contact point for anyone interested in this work.
```

**Review by bradh — COMMENTED** [review]
LGTM.

**Pull request merged**